### ISSUE TITLE
Sync issue with in github actions

### ISSUE THREAD

**Issue by davidferlay** [skilld-labs/plasmactl-bump#33]
### Observed

- While adding CI testing to plasmactl-bump github repo, I'm facing this strange issue:
   - https://github.com/skilld-labs/plasmactl-bump/actions/runs/9221674797/job/25371217117
   - You can see executed code here: https://github.com/skilld-labs/plasmactl-bump/blob/ciupdate/.github/workflows/commit.yml#L22-L33

It's not reproduced locally nor on Gitlab CI, so it seems to be an issue only with github actions, for some reasons.

Maybe something related to how git repo is handled here

### Expected

- If there is a way to make sync work in Gitbub action, let's do it as it will be cool to have all commands covered by upstream CI testing

**Comment by davidferlay** [comment]
Added CI check  to same branch as fix to see if it passes or not
- before: https://github.com/skilld-labs/plasmactl-bump/actions/runs/9483673652/job/26131313513?pr=32
- after: https://github.com/skilld-labs/plasmactl-bump/actions/runs/9483683274/job/26131349840?pr=34

So another issue appeared:
```
Error: no remote was found
```
it seems that repo name is not found by code while running in github action CI
